services/docker/generate.go

```diff
@@ -94,27 +95,29 @@ func MarshalledDefaults() []byte {
 // UnmarshalledDefaults ...
 func UnmarshalledDefaults() conf.ComposeDotTok {
 	tokStruct := conf.ComposeDotTok{}
-	xdebugImageVersion := conf.GetConfig().Tokaido.Stability
 	phpVersion := conf.GetConfig().Tokaido.Phpversion
+	unisonVersion := version.GetUnisonVersion()
```

Comment:
Could this version check be moved into the switch case for unison?

Reply:
also a good point - fixed thanks